lib/ts/recipe/emailverification/api/implementation.ts

```diff
@@ -73,7 +76,7 @@ export default function getAPIInterface(): APIInterface {
 
             let userId = session.getUserId();
```

Comment:
Not exactly related to this PR, but since we are modifying these APIs, one of the other pending tasks was that the session object should be an input to this function. This is cause if the user wants to override this API and do something with the session, at the moment, they will have to call `getSession` (so this function would be called twice, which is kind of weird). So adding this as an input to this function allows them to access the session container easily.